packetbeat/protos/rpc/nfs4.go

```diff
@@ -0,0 +1,239 @@
+package rpc
+
+import "fmt"
+
+const OP_ACCESS = 3
```

Comment:
Consider grouping these constants in a single `const` block. It's common to group related constants this way.

```
const (
    OP_ACCESS = 3
    OP_CLOSE = 4
)
```

I think you could also use [`iota`](https://golang.org/ref/spec#Iota) here.

```
const (
    OP_ACCESS = iota + 3
    OP_CLOSE
    OP_COMMIT
    ...
    OP_ILLEGAL = 10044
)
```